### PR TITLE
Patch for Ember 2.8.0

### PR DESCRIPTION
**Pull request by aptran**
Patch to make compatible with Ember 2.8. 
Cannot observe `parentView`.

https://github.com/emberjs/ember.js/issues/14266
https://github.com/emberjs/ember.js/issues/14260

### CODE REVIEW

File: addon/mixins/finds-parent-form-view.js [hunk 1]
```diff
@@ -2,13 +2,17 @@ import Ember from 'ember';
...
 export default Ember.Mixin.create({
 
-  parentFormView: Ember.computed('parentView', function () {
-    var parentView = this;
-    while (parentView = parentView.get('parentView')) {
-      if (parentView.get('tagName') === 'form') return parentView;
-    }
-  }),
+  parentFormView: Ember.computed(function () {
+    parentFormViewMacro(this.get('parentView'));
```
Comment (jakiestfu) [inline addon/mixins/finds-parent-form-view.js]: !fix change to `getFormView`

Comment (aptran) [reply addon/mixins/finds-parent-form-view.js]: !fixed

**Comment by briangonzalez** [comment]
+1

**Comment by aptran** [comment]
Closing this PR since these changes were included in `fixes-and-upgrades`

**Pull request closed without merging**